_principles/index.md

```diff
@@ -33,15 +33,15 @@ In this document, we’ll explain each idea and how it will influence our work t
 
 # Methodologies
 
-## <a name="letting-users-guide-our-work"></a>Letting users guide our work
```

Comment:
Jekyll adds landmark IDs to headings automatically, so we don't need to do it explicitly with these anchors. Plus the `name` attribute is deprecated now.